torch/csrc/jit/codegen/cuda/ir_nodes.cpp

```diff
@@ -569,9 +569,18 @@ Val* IterDomain::extent() const {
 void IterDomain::parallelize(ParallelType t) {
   parallel_type_ = t;
 
-  TORCH_CHECK(t != ParallelType::Vectorize, "Vectorization not yet supported.");
+  if (t == ParallelType::Vectorize) {
+    TORCH_CHECK(
```

Comment:
Are there limitations for the generated code on word size vs number of vectorized elements supported? What about contiguity of the tensor, and which iter domains can actually be vectorized. I assume we can't do strided vectorized loads/stores.

Reply:
Are there also restrictions on being within the compute at position?

Reply:
I didn't add any checks / restrictions yet.